pkg/grafana-organization-reconciler/grafana_organization_reconciler.go

```diff
@@ -0,0 +1,77 @@
+package grafanaorganizationreconciler
+
+import (
+	"context"
+
+	"github.com/giantswarm/observability-operator/api/v1alpha1"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/cluster-api/util/patch"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/giantswarm/logging-operator/pkg/key"
+	"github.com/giantswarm/logging-operator/pkg/reconciler"
+)
+
+// GrafanaOrganizationReconciler reconciles grafanaOrganization CRs
+type GrafanaOrganizationReconciler struct {
+	client.Client
+	Scheme      *runtime.Scheme
+	Reconcilers []reconciler.Interface
+}
+
+func (l *GrafanaOrganizationReconciler) Reconcile(ctx context.Context) (result ctrl.Result, err error) {
+
+	return result, errors.WithStack(err)
+}
+
+func (l *GrafanaOrganizationReconciler) reconcileCreate(ctx context.Context, grafanaOrganization v1alpha1.GrafanaOrganization) (ctrl.Result, error) {
+	logger := log.FromContext(ctx)
+
+	if !controllerutil.ContainsFinalizer(&grafanaOrganization, key.Finalizer) {
```

Comment:
Can you remind me what this finalizer is?

Reply:
It's the operator's finalizer